dashboard/app/views/home/teacher_homepage.html.haml

```diff
@@ -1,4 +1,21 @@
+:ruby
 
-#container
+  courses = current_user.in_progress_and_completed_scripts.slice(0,2).map do |script|
+    script_id = script[:script_id]
+    script_name = Script.find_by_id(script_id)[:name]
+    {
+      id: script_id,
+      script_name: script_name,
+      name: data_t_suffix('script.name', script_name, 'title'),
+      description: data_t_suffix('script.name', script_name, 'description_short'),
+      link: script_url(script_id),
+      image: ""
+    }
```

Comment:
Another thing to note is that the "proper" way to do this is probably not to have this logic live in the HAML file (which should just be view logic), but instead to live in the model or controller (and have unit tests written for it). Given that this is temporary until the course stuff is ready, I think doing it this way is fine for now.